pod/perllocale.pod

```diff
@@ -588,15 +610,17 @@ and see whether they list something resembling these
 Sadly, even though the calling interface for C<setlocale()> has been
 standardized, names of locales and the directories where the
 configuration resides have not been.  The basic form of the name is
-I<language_territory>B<.>I<codeset>, but the latter parts after
-I<language> are not always present.  The I<language> and I<country>
+I<language[_territory[.codeset]][@modifier]>.
+The I<language> and I<country>
 are usually from the standards B<ISO 3166> and B<ISO 639>, the
 two-letter abbreviations for the countries and the languages of the
 world, respectively.  The I<codeset> part often mentions some B<ISO
 8859> character set, the Latin codesets.  For example, C<ISO 8859-1>
 is the so-called "Western European codeset" that can be used to encode
 most Western European languages adequately.  Again, there are several
 ways to write even the name of that one standard.  Lamentably.
+I<modifier> is very individualized to the rest of the locale, naming
+some variant, such as a different currency symbol than otherwise.
```

Comment:
I don't quite get the two lines above.  Would it be okay to use `for example,` instead of `such as`?  Should `otherwise` be interpreted to mean `one would normally use`?

Reply:
I tried `for example`, but found the result ambiguous.  How does the revised version seem to you @jkeenan

Reply:
That's better.  Ship it.